inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php

```diff
@@ -1010,6 +1018,8 @@ public function process_on_submit_jobs() {
 		foreach ( $rows as $row ) {
 			$response = $this->send_api( $row->url, (bool) $row->is_mobile );
 			if ( false === $response || ! isset( $response['contents'], $response['contents']['jobId'], $response['contents']['queueName'] ) ) {
+
+				$this->used_css_query->make_status_failed( (int) $row->id, '',  '' );
```

Comment:
Please store response['code'] and response['message'] if available in the DB.

Reply:
Ah right... OK, let's keep it like this for now and open an issue for later to store the response in case of failure when adding to the queue. OK.